tools/dockerfile/grpc_artifact_python_linux_armv7/Dockerfile

```diff
@@ -62,5 +66,25 @@ ENV CXX ${CROSS_TRIPLE}-g++
 # TODO(jtattermusch): Remove this hack when possible.
 RUN echo "# file contents removed to avoid resetting PATH set by the docker image" >/etc/profile
 
+# TODO: simplify the list of third_party modules list
+# NOTE: git>=2.46 allows leading paths like third_party/* to include all subdirectories
+# current docker base images use git versions lower than 2.46 and hence require separate configs for each submodule  
 RUN git config --global --add safe.directory /var/local/jenkins/grpc
+RUN git config --global --add safe.directory /var/local/jenkins/grpc/.git
```

Comment:
This is not an immediate blocker, but please take an action item to remove the explicit enumeration of third-party modules. This is going to make it significantly more difficult to add a third-party module in the future.

Reply:
This change will only be needed on the master branch.